test/Microsoft.AspNet.Mvc.ModelBinding.Test/Metadata/ModelMetadataProviderTest.cs

```diff
@@ -898,7 +898,7 @@ public AttributeInjectModelMetadataProvider(object[] attributes)
                 _attributes = attributes;
             }
 
-            protected override DefaultMetadataDetails CreateTypeDetails([NotNull]ModelMetadataIdentity key)
+            protected override DefaultMetadataDetails CreateTypeDetails(ModelMetadataIdentity key)
```

Comment:
Will revert, however I think our guidance  has been that the interface has `[NotNull]` and the class can skip it. Is that not the case?